caesars-palace/blocks/carousel/carousel.js

```diff
@@ -374,6 +374,16 @@ export default async function decorate(block) {
       block.querySelectorAll('img').forEach((image) => {
         image.closest('picture').replaceWith(createOptimizedPicture(image.src, image.alt, false, [{ width: '768' }]));
       });
+
+      const firstSlide = document.getElementById('carousel-slide1');
+      firstSlide.querySelectorAll('img').forEach((image) => {
+        image.loading = 'eager';
+      });
```

Comment:
Could this be turned into a function to reduce duplication?

eg.
```
function applyEagerLoading(slide) {
      if (!slide) {
        return;
      }
      slide.querySelectorAll('img').forEach((image) => {
        image.loading = 'eager';
      });
}

applyEagerLoading(document.getElementById('carousel-slide1'));
```

Reply:
Instead of `document.getElementById` you could also try `block.querySelector('#carousel-slide1')`. 

A quick search does seem to indicate `getElementById` is faster but not perceptible.

https://gomakethings.com/javascript-selector-performance/

Reply:
Any reason to do that? Ids should be unique